modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,34 +6,69 @@
 [id="persistent-storage-csi-migration-sc-vsphere_{context}"]
 = vSphere automatic migration
 
+[IMPORTANT]
+====
+Migration includes significant consequences:
+
+* Enabling migration cannot be undone.
+
+* Migration can take awhile to complete depending on how many nodes are on the cluster.
+====
+
 == New installations of {product-title}
 For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
+== Upgrading from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.13 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} upgrade can occur and automatic migration is enabled by default.
```

Comment:
```suggestion
If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.13 to 4.14, first (?) update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, you can upgrade your {product-title} cluster and automatic migration is enabled by default.
```